resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -288,9 +310,16 @@ class VisualEffect
             {
                 New-Item -Path $global:ControlPanelAccessibilityRegistryPath -Force | Out-Null
             }
+            if (-not (Test-Path -Path $global:ControlPanelAccessibilityRegistryPath))

```

Comment:
Is this the wrong registry path? Please make sure that you verify the behavior of these tests manually if needed.